src/components/toast/toast/toast.module.scss

```diff
@@ -0,0 +1,74 @@
+@import '@styles/variables.scss';
+
+.root {
+    padding: 14px;
+    width: 480px;
+    color: var(--white);
+    display: flex;
+    justify-content: space-between;
+    align-items: start;
+    gap: 14px;
+}
+
+.root.error {
+    background-color: var(--error);
+}
+
+.root.success {
+    background-color: var(--colorE4);
+}
+
+.root.blank {
+    background-color: var(--colorC3);
+}
+
+.root.loading {
+    width: 280px;
+    text-align: center;
+    background-color: var(--secondary1);
+    color: var(--secondary4);
+}
+
+@media (max-width: $desktop-width) {
+    .root,
+    .root.loading {
+        width: 100%;
+    }
+}
+
+.description {
+    font: var(--paragraph3);
+    flex-grow: 1;
+}
+
+.closeButton {
+    flex-shrink: 0;
+    display: flex;
+    cursor: pointer;
+}
+
+@keyframes enterKeyframes {
```

Comment:
```suggestion
@keyframes fade-in {
```

Reply:
I'm thinking what if user will change `opacity` to `background-color` or `transform`? Then they should change fade-in / fade-out names.